next/graphql/queries/General.graphql

```diff
@@ -0,0 +1,63 @@
+query Footer($locale: I18NLocaleCode!) {
+  footer(locale: $locale) {
+    ...Footer
+  }
+}
+
+fragment Footer on FooterEntityResponse {
```

Comment:
Consider if `FooterFragment` won't be misinterpreted as fragment `on Footer` while it is on `FooterEntityResponse`. We used to do fragments as `Footer on Footer` and `FooterEntity on FooterEntity`.